chart/values.yaml

```diff
@@ -392,6 +374,7 @@ components:
       nodeRoots: 
       - /var/lib
       - /run/containerd/io.containerd.runtime.v1.linux/k8s.io
+      #- /run/containerd/io.containerd.runtime.v2.task/k8s.io
```

Comment:
What are others to do with this comment? If this isn't a leftover, please add some comment what this is for.